india_compliance/gst_india/doctype/gstin/gstin.py

```diff
@@ -134,7 +134,7 @@ def get_and_validate_gstin_status(gstin, doc):
 
 
 @frappe.whitelist()
-def get_gstin_status(gstin, doc=None, force_update=False):
+def get_gstin_status(gstin, doc=None, force_update: bool = False):
```

Comment:
@coderabbitai

Issue: Missing type hints caused JS booleans to arrive as Python strings ("true"/"false").
Fix: Added explicit type annotations to all pre-pre whitelisted methods in India compliance.
Result: Correct type deserialization (bool/int/float/str), fewer logic errors, better IDE/CI checks.

Always check that a type hint for bool param in whitelisted method is present. Ensure this for all frappe apps.